### DISCUSSION TITLE
Tailscale

### DISCUSSION THREAD

**Discussion by akula3** [OpenCCU/OpenCCU#1854] [🇩🇪 Fragen&Antworten] [answered]
Moin-moin!
Ich habe neulich ein Schreck bekommen - meine RaspberryMatic war nicht mehr erreichbar! Weder aus dem Heimnetzwek noch über Cloudmatic oder SSH. Obwohl die Automatisationen und Programme liefen dabei anscheinend ordnungsgemäß und ich habe die RaspberryMatic auch in der FritzBox mit der gewohnten IP Adresse gesehen. Ich habe dann die RaspberryMatic zwei Mal vom Strom genommen und wieder hochgefahren - ohne Erfolg.
Der Grund dafür war die Anmeldung der RaspberryMatic bei Tailscale. Ab dem Zeitpunktz der Anmeldung war sie nur noch unter der von Tailscale vergebenen IP erreichbar - egal, ob von außen oder aus dem Heimnetzwerk. **Ob das so gewollt ist?**
Der HomeAssistant auf separaten RPi ist bei mir auch bei Tailscale angemeldet, trotzdem ist er im Heimnetzwerk unter der normalen lokalen IP Adresse erreichbar.
Es gibt auch kein Hinweis für dieses Verhalten in der Wiki. Wäre klasse, wenn es dort aufgenommen wäre, damit andere User bescheid wissen.
Viele Grüße,
Konstantin

**Comment by Baxxy13** [comment, marked as answer]
Naja, es ist nicht vorgesehen 2 Geräte als "Tailscale-Server" in einem Netzwerk zu betreiben.
Wenn auf HA Tailscale aktiv ist solltest du doch von extern auch Zugriff auf alle deine Heimnetzgeräte (inkl. RaspberryMatic) haben.
Da braucht man auf RM Tailscale nicht auch noch aktivieren.

Das Problem ist das Tailscale in HA sowie in RM per default so konfiguriert ist das du damit Zugriff auf alle Heimnetzgeräte hast.
Dazu dient die Option: `--advertise-routes` Wenn bei 2 oder mehr Geräten im selben Netz die Option aktiviert ist entstehen die Probleme die du hattest. Da beißt sich dann irgendwas.

**Comment by akula3** [comment]
> Wenn auf HA Tailscale aktiv ist solltest du doch von extern auch Zugriff auf alle deine Heimnetzgeräte (inkl. RaspberryMatic) haben.> 

Du hast natürlich recht, so hatte ich auch im Endeffekt eingerichtet.
Ich hatte nur zur Probe den Tailscale auf RM aktiviert.
